tests/pointer.c

```diff
@@ -42,13 +42,21 @@ int t5()
     expect(65, *s);
 }
 
+int t6()
+{
+    void *x = 1;
```

Comment:
Avoid assigning pointer value to `1` since the address is usually reserved for certain purpose. Instead, you can refer to some initiated object.

Reply:
Not resolved yet.